WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsDetails.swift

```diff
@@ -1,19 +1,28 @@
 import SwiftUI
 
-struct OrderCustomFieldsDetails: View {
+struct CustomFieldsDetails: View {
```

Comment:
Nit: Please rename the view to clarify that it's a view. The postfix `View` is usually used for this.

Reply:
Sure, updated in https://github.com/woocommerce/woocommerce-ios/pull/13639/commits/27ec9b5beb935f8f34f492ceb2e5ac5acfb6761b